assets/js/admin/wc-shipping-zone-methods.js

```diff
@@ -138,7 +138,7 @@
 							var $tr = view.$el.find( 'tr[data-id="' + rowData.instance_id + '"]');
 
 							if ( ! rowData.has_settings ) {
-								$tr.find( '.wc-shipping-zone-method-title a').replaceWith( $tr.find( '.wc-shipping-zone-method-title' ).text() );
+								$tr.find(".wc-shipping-zone-method-title").text(t.find(".wc-shipping-zone-method-title > a").text() );
```

Comment:
The code style on this line doesn't match the WP style guides. Single quotes should be used and spaces should be between function parentheses. 

See the previous version for an example of how the code style should look: `$tr.find( '.wc-shipping-zone-method-title a' ).replaceWith( $tr.find( '.wc-shipping-zone-method-title' ).text() );`